### PR TITLE
[WIP] Improve reference to CLI and preview

### PR DESCRIPTION
**Pull request by radeva**
- Improve reference to CLI and `tns preview`
- remove any references to Sidekick

### CODE REVIEW

File: cleanWeather-vue/1.md [hunk 1]
```diff
@@ -60,6 +60,6 @@ One of the biggest benefits of using NativeScript is the fast development workfl
 
 > **TIP**:
 > * You can save your current Playground work by clicking the **Save** button at the top. This will generate a URL that you can bookmark for future use.
-> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue developing using [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/start/quick-setup).
+> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue by installing NativeScript command-line interface with `npm i nativescript -g` and running `tns preview` command to preview your work the same way you do it in Playground.
```
Comment (ggarabedian) [inline cleanWeather-vue/1.md]: [Suggestion] You can consider rephrasing this to:
After you download the app, you should install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.

File: getting-started-ng/1.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ One of the biggest benefits of using NativeScript is the fast development workfl
 
 > **TIP**:
 > * You can save your current Playground work by clicking the **Save** button at the top. This will generate a URL that you can bookmark for future use.
-> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue developing using [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/start/quick-setup).
+> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue by installing NativeScript command-line interface with `npm i nativescript -g` and running `tns preview` command to preview your work the same way you do it in Playground.
```
Comment (ggarabedian) [inline getting-started-ng/1.md]: [Suggestion] Same as above

File: getting-started-ng/success.md [hunk 3]
```diff
@@ -1,7 +1,12 @@
-Now that you’ve learned the NativeScript basics, you’re ready to move on to our next tutorial, which has you built a complete iOS and Android app from scratch.
+Now that you’ve learned the NativeScript basics, you’re ready to move on to our next tutorial, which has you built a complete iOS and Android app from scratch. To continue you can:
```
Comment (ggarabedian) [inline getting-started-ng/success.md]: "(...), which has you built a (...)" doesn't sound right to me. Consider replacing it with:
1. "(...) in which you get to build a complete (...)" 
2. "(...) in which you'll see how to build a complete (...)"

File: getting-started-ng/success.md [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
 
-You can complete that tutorial in Playground, or, if you prefer to work in a local development environment, you can continue in [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/angular/tutorial/ng-chapter-1).
+* **Option 1. Complete the next tutorial in Playground**  
+Complete <a id="btn-next-tutorial" href="/?template=groceries-ng&tutorial=groceries-ng"><strong>Build a Full App</strong>&nbsp;tutorial in Playground</a>
```
Comment (ggarabedian) [inline getting-started-ng/success.md]: Complete **the** Build a Full App tutorial in Playground.

File: getting-started-ng/success.md [hunk 5]
```diff
@@ -1,7 +1,12 @@
...
 
-As always, if you get stuck, feel free to reach for help on [**Stack Overflow**](https://stackoverflow.com/questions/tagged/nativescript) or ask our experts in [**NativeScript community Slack channel**](http://developer.telerik.com/wp-login.php?action=slack-invitation). Have fun building with NativeScript!
+* **Option 2. Start working in a local environment with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install NativeScript CLI by running `npm i nativescript -g` and use `tns preview` command to preview you app on real device the same way you do it in Playground.
```
Comment (ggarabedian) [inline getting-started-ng/success.md]: [Suggestion] Add "**the**" in front of NativeScript CLI ("(...) you can install **the** NativeScript CLI (...)"). Also, consider adding "**the**" in front of "`tns preview` command".

File: getting-started-ng/success.md [hunk 6]
```diff
@@ -1,7 +1,12 @@
...
+* **Option 2. Start working in a local environment with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install NativeScript CLI by running `npm i nativescript -g` and use `tns preview` command to preview you app on real device the same way you do it in Playground. 
+
+* **Option 3. Start working in a local environment with NativeScript CLI & no limitations**  
+If you need to build a production app, you can [do a full install of NativeScript](https://docs.nativescript.org/angular/start/quick-setup) and start working without any limitations. The NativeScript CLI lets you build full native iOS and Android apps locally on your development machine.
 
-<a id="btn-next-tutorial" href="/?template=groceries-ng&tutorial=groceries-ng">Start the&nbsp;<strong>Build a Full App</strong>&nbsp;tutorial</a>
+As always, if you get stuck, feel free to reach for help on [**Stack Overflow**](https://stackoverflow.com/questions/tagged/nativescript) or ask our experts in [**NativeScript community Slack channel**](http://developer.telerik.com/wp-login.php?action=slack-invitation). Have fun building with NativeScript!
```
Comment (ggarabedian) [inline getting-started-ng/success.md]: [Suggestion] (...) or ask our experts in **the** [**NativeScript community Slack channel**] (...)

File: groceries-js/success.md [hunk 7]
```diff
@@ -1,11 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install NativeScript CLI by running `npm i nativescript -g` and use `tns preview` command to preview you app on real device the same way you do it in Playground.
```
Comment (ggarabedian) [inline groceries-js/success.md]: Same as above

File: groceries-ng/success.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install NativeScript CLI by running `npm i nativescript -g` and use `tns preview` command to preview you app on real device the same way you do it in Playground.
```
Comment (ggarabedian) [inline groceries-ng/success.md]: Same as above :)

File: cleanWeather-vue/1.md [hunk 9]
```diff
@@ -60,6 +60,6 @@ One of the biggest benefits of using NativeScript is the fast development workfl
 
 > **TIP**:
 > * You can save your current Playground work by clicking the **Save** button at the top. This will generate a URL that you can bookmark for future use.
-> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue developing using [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/start/quick-setup).
+> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app, you should install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
```
Comment (tjvantoll) [inline cleanWeather-vue/1.md]:
```suggestion
> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app, you should install the NativeScript CLI by running the `npm i nativescript -g` command, and then execute the `tns preview` command to preview your work—just as you do in Playground.
```

File: getting-started-ng/1.md [hunk 10]
```diff
@@ -78,7 +78,7 @@ One of the biggest benefits of using NativeScript is the fast development workfl
 
 > **TIP**:
 > * You can save your current Playground work by clicking the **Save** button at the top. This will generate a URL that you can bookmark for future use.
-> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue developing using [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/start/quick-setup).
+> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app, you should install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
```
Comment (tjvantoll) [inline getting-started-ng/1.md]:
```suggestion
> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app, you should install the NativeScript CLI by running the `npm i nativescript -g` command, and then execute the `tns preview` command to preview your work—just as you do in Playground.
```

File: groceries-js/success.md [hunk 11]
```diff
@@ -1,11 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**
```
Comment (tjvantoll) [inline groceries-js/success.md]:
```suggestion
* **Option 1. Continue with NativeScript CLI & Playground Preview App**  
```

Comment (tjvantoll) [reply groceries-js/success.md]: :)

File: groceries-js/success.md [hunk 12]
```diff
@@ -1,11 +1,10 @@
...
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
 
-* **[Option 2. Install the NativeScript CLI](https://docs.nativescript.org/angular/tutorial/ng-chapter-1)**  
-If you’re comfortable working with a command-line interface, you can install and work with the NativeScript CLI. The NativeScript CLI lets you build full iOS and Android apps locally on your development machine.
-
-As always, if you get stuck, feel free to reach for help on [**Stack Overflow**](https://stackoverflow.com/questions/tagged/nativescript) or ask our experts in [**NativeScript community Slack channel**](http://developer.telerik.com/wp-login.php?action=slack-invitation). Have fun building with NativeScript!
+* **Option 3. Continue with NativeScript CLI & no limitations**
```
Comment (tjvantoll) [inline groceries-js/success.md]:
```suggestion
* **Option 2. Continue with NativeScript CLI & no limitations**  
```

File: groceries-js/success.md [hunk 13]
```diff
@@ -1,11 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
```
Comment (tjvantoll) [inline groceries-js/success.md]:
```suggestion
If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command, and then execute the `tns preview` command to preview your work—just as you do in Playground.
```

File: getting-started-ng/success.md [hunk 14]
```diff
@@ -1,7 +1,12 @@
...
 
-You can complete that tutorial in Playground, or, if you prefer to work in a local development environment, you can continue in [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/angular/tutorial/ng-chapter-1).
+* **Option 1. Complete the next tutorial in Playground**  
+Complete the <a id="btn-next-tutorial" href="/?template=groceries-ng&tutorial=groceries-ng"><strong>Build a Full App</strong>&nbsp;tutorial in Playground</a>
 
-As always, if you get stuck, feel free to reach for help on [**Stack Overflow**](https://stackoverflow.com/questions/tagged/nativescript) or ask our experts in [**NativeScript community Slack channel**](http://developer.telerik.com/wp-login.php?action=slack-invitation). Have fun building with NativeScript!
+* **Option 2. Start working in a local environment with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
```
Comment (tjvantoll) [inline getting-started-ng/success.md]:
```suggestion
If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command, and then execute the `tns preview` command to preview your work—just as you do in Playground.
```

File: groceries-ng/success.md [hunk 15]
```diff
@@ -1,10 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**
```
Comment (tjvantoll) [inline groceries-ng/success.md]:
```suggestion
* **Option 1. Continue with NativeScript CLI & Playground Preview App**  
```

File: groceries-ng/success.md [hunk 16]
```diff
@@ -1,10 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
 
-* **[Option 2. Install the NativeScript CLI](https://docs.nativescript.org/angular/tutorial/ng-chapter-1)**  
-If you’re comfortable working with a command-line interface, you can install and work with the NativeScript CLI. The NativeScript CLI lets you build full iOS and Android apps locally on your development machine.
+* **Option 3. Continue with NativeScript CLI & no limitations**
```
Comment (tjvantoll) [inline groceries-ng/success.md]:
```suggestion
* **Option 2. Continue with NativeScript CLI & no limitations**  
```

File: groceries-ng/success.md [hunk 17]
```diff
@@ -1,10 +1,10 @@
 Now that you’ve learned the NativeScript basics, you’re ready to set up a professional development environment to build your apps.
 
-* **[Option 1. Download NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick)**  
-NativeScript Sidekick is a desktop development tool for building NativeScript apps. Sidekick offers starter templates to help you get started,as well as tooling for more advanced NativeScript functionality, such as plugins, builds, and debugging.
+* **Option 2. Continue with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.
```
Comment (tjvantoll) [inline groceries-ng/success.md]:
```suggestion
If you’re comfortable working with a command-line interface, you can install the NativeScript CLI by running the `npm i nativescript -g` command, and then execute the `tns preview` command to preview your work—just as you do in Playground.
```

**Pull request merged**